pytest_tests/tests/network/test_deployment.py

```diff
@@ -0,0 +1,94 @@
+import time
+from importlib.resources import files
+
+import allure
+import pytest
+import yaml
+from helpers.common import SIMPLE_OBJECT_SIZE
+from helpers.file_helper import generate_file, get_file_hash
+from helpers.neofs_verbs import get_object_from_random_node, put_object_to_random_node
+from helpers.wallet_helpers import create_wallet
+from neofs_testlib.env.env import NeoFSEnv, NodeWallet
+
+
+@pytest.fixture
+def clear_neofs_env():
+    neofs_env_config = yaml.safe_load(
+        files("neofs_testlib.env.templates").joinpath("neofs_env_config.yaml").read_text()
+    )
+    neofs_env = NeoFSEnv(neofs_env_config=neofs_env_config)
+    yield neofs_env
+    neofs_env.kill()
+
+
+def put_get_object(neofs_env: NeoFSEnv, wallet: NodeWallet):
+    cli = neofs_env.neofs_cli(neofs_env.generate_cli_config(wallet))
+
+    result = cli.container.create(
+        rpc_endpoint=neofs_env.sn_rpc,
+        wallet=wallet.path,
+        policy="REP 3 IN X CBF 1 SELECT 1 FROM * AS X",
```

Comment:
Does it work this way with `SELECT 1`? @carpawell?

Reply:
fixed, my bad, needed to have select 3.